guides/GuideForGuests.md

```diff
@@ -0,0 +1,88 @@
+
+* Your name
+* Name of the project
+* Project website(s)
+* 1 sentence about the project
+* 1 sentence about how someone can get involved (including contact information)
+
+The structure of the blog will be three introductory sentences, the video, and then bullet points with the information above for the three guest projects. We'd like to turn this around very quickly, so please try to provide the sentences in advance rather than after the call. Here's the google form link again: https://goo.gl/forms/3foIuNP9G7kmCW193
+
+### During the call
+
+***Please join the call 5 minutes before the offical start time so we're all ready to go!***
+
+The call will start with a brief welcome, and then we'll jump into the conversations. These should last between 8 and 10 minutes. Your host will contact you in advance to discuss the running order.
+
+#### Sharing your screen
+
+It's very easy to share your screen from within Google Hangouts, and this might be a really great way to orient our viewers to your project. You could navigate to the project webpage, or showcase an example.
+
+You do not need to prepare a presentation for this call. In fact, we'd prefer that you didn't! It will be much more fun to see your face when you're describing the project. We're confident that you know an awful lot about it and don't need the slides. However, if there are some images that would particularly help to describe your project goals to our audience, then feel free to have them ready to share on your screen.
+
+#### Additional guests
+
+It is very likely that your project is a collaborative endeavor. If you'd like to invite other representatives of the project to the call that's absolutely fine, just remember that we don't have a huge amount of time, so probably more than three people per project will be difficult to wrangle efficiently.
+
+The additional guests can either be in the same place as you (and therefore connect on the same screen) or they can join the google hangout individually.
+
+#### Asking and answering questions
+
+Google Hangouts does not allow more than 10 people to join a call at one time, which is why our audience will mostly be watching on YouTube and asking questions via live chat. You'll be there the whole time, so ***please join in to ask questions of your fellow guests***. You do not need to sit silently and watch Kirstie chat with each guest in turn! You're very welcome to ask your own questions :smiley:
+
+Although Kirstie will try to monitor the live chat, it would also be very helpful if the guests who are not currently being interviewed could monitor the chat and verbalise those questions on the hangout. You don't have to, but multi-tasking is sometimes a tough job and any assistance will be greatly appreciated.
```

Comment:
This brings up another point-- I highly doubt this would happen, but is there a mechanism to block abusive / troll comments on the live chat?

It looks like one way to do this would be to designate moderators and/or enable "Hold for review" https://support.google.com/youtube/answer/2524549?hl=en

Reply:
Added a sentence about the moderator. Need to add a link to the CoC when it's in the repo 😄